src/instruments/src/PFD/FMA.tsx

```diff
@@ -537,6 +546,82 @@ const BC1Cell = () => {
     );
 };
 
+const getBC3Message = (isAttExcessive: boolean) => {
+    const armedVerticalBitmask = getSimVar('L:A32NX_FMA_VERTICAL_ARMED', 'number');
+    const TCASArmed = (armedVerticalBitmask >> 6) & 1;
+
+    const trkFpaDeselectedTCAS = getSimVar('L:A32NX_AUTOPILOT_TCAS_MESSAGE_TRK_FPA_DESELECTION', 'bool');
+    const tcasRaInhibited = getSimVar('L:A32NX_AUTOPILOT_TCAS_MESSAGE_RA_INHIBITED', 'bool');
+
+    let text: string;
+    let className: string;
+    // All currently unused message are set to false
+    if (false) {
+        text = 'MAN PITCH TRIM ONLY';
+        className = 'Red Blink9Seconds';
+    } else if (false) {
+        text = 'USE MAN PITCH TRIM';
+        className = 'PulseAmber9Seconds Amber';
+    } else if (false) {
+        text = 'FOR GA: SET TOGA';
+        className = 'PulseAmber9Seconds Amber';
+    } else if (TCASArmed && !isAttExcessive) {
+        text = '  TCAS                ';
+        className = 'Cyan';
+    } else if (false) {
+        text = 'DISCONNECT AP FOR LDG';
+        className = 'PulseAmber9Seconds Amber';
+    } else if (tcasRaInhibited && !isAttExcessive) {
+        text = 'TCAS RA INHIBITED';
+        className = 'White';
+    } else if (trkFpaDeselectedTCAS && !isAttExcessive) {
+        text = 'TRK FPA DESELECTED';
+        className = 'White';
+    } else if (false) {
+        text = 'SET GREEN DOT SPEED';
+        className = 'White';
+    } else if (false) {
+        text = 'T/D REACHED';
+        className = 'White';
+    } else if (false) {
+        text = 'MORE DRAG';
+        className = 'White';
+    } else if (false) {
+        text = 'CHECK SPEED MODE';
+        className = 'White';
+    } else if (false) {
+        text = 'CHECK APPR SELECTION';
+        className = 'White';
+    } else if (false) {
+        text = 'TURN AREA EXCEEDANCE';
+        className = 'White';
+    } else if (false) {
+        text = 'SET HOLD SPEED';
+        className = 'White';
+    } else if (false) {
+        text = 'VERT DISCONT AHEAD';
+        className = 'Amber';
+    } else if (false) {
+        text = 'FINAL APP SELECTED';
+        className = 'White';
+    } else {
```

Comment:
This would be better off using the new message framework from CFPM (which is intended for MCDU, ND and PFD). There is at least no need to add any FM messages as they will be handled by that system.

Reply:
How should we move forward? Right now it's integrated with AP in the way above. Can we do that with an upcoming PR?